UnityProject/Assets/Scripts/Items/PDA/PDALogic.cs

```diff
@@ -436,7 +436,7 @@ public void SpawnUplinkItem(GameObject objectRequested, int cost)
 
 			if (cost > UplinkTC) return;
 
-			var result = Spawn.ServerPrefab(objectRequested);
```

Comment:
humm, was going to suggest replacement strategy but then realise it doesn't return any slots, so this is the best thing for now